public/en/releases.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
```

Comment:
Do we have some kind of html style guide we use? I'm asking because an uppercase doctype feels inconsistent with lowercasing the rest of the html tags.